engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -556,23 +556,26 @@ private Duration getTemporalValidationTolerance(ConfigurationState configuration
 		if ( configurationState instanceof ConfigurationImpl ) {
 			ConfigurationImpl hibernateSpecificConfig = (ConfigurationImpl) configurationState;
 			if ( hibernateSpecificConfig.getTemporalValidationTolerance() != null ) {
-				LOG.logTemporalValidationTolerance( hibernateSpecificConfig.getTemporalValidationTolerance() );
+				if ( !Duration.ZERO.equals( hibernateSpecificConfig.getTemporalValidationTolerance() ) ) {
+					LOG.logTemporalValidationTolerance( hibernateSpecificConfig.getTemporalValidationTolerance() );
+				}
```

Comment:
Is it null if it hasn't been defined at all? If so, maybe we should consider logging it anyway as it has been explicitly set and only remove the `LOG.logTemporalValidationTolerance( Duration.ZERO );` line below?

Reply:
Yes, I actually also thought about this while walking home - we should log the value even if it's 0 but it was explicitly defined. Cleaned up the commit and forced pushed.